packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -37,6 +37,7 @@ interface MergeConfig extends Omit<Partial<TestrunnerOptionsWithParameters>, 'sp
 }
 
 export default class ConfigParser {
+    #ignoreSpec = false
```

Comment:
Can we add a comment mentioning for what this is being used? I can see that in some month this will be difficult to understand

Reply:
Added